src/CitizenProject/CitizenProjectCreationCommandHandler.php

```diff
@@ -39,6 +39,9 @@ public function handle(CitizenProjectCreationCommand $command, TurnkeyProject $t
             $this->citizenProjectManager->addImage($citizenProject);
         } elseif ($turnkeyProject) {
             $this->citizenProjectManager->copyImageFromTurnkeyProject($citizenProject, $turnkeyProject);
+        } else {
+            $citizenProject->setImageName(CitizenProjectManager::CITIZEN_PROJECT_DEFAULT_IMAGE_NAME);
```

Comment:
I liked the solution with CP manager, because we incapsulate this logic in the class 
```
$this->citizenProjectManager->setDefaultImage($citizenProject);
```